File: php/testcode/benchmark_test_00831.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00831(BenchmarkRequest $req): BenchmarkResponse {
    $url = $req->param('url');
    if (parse_url($url, PHP_URL_HOST) === 'example.com') {
        header('Location: ' . urldecode($url));
    }
    return BenchmarkResponse::ok('');
}

File: rust/testcode/benchmark_test_00831.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let url = req.param("url");
    let ctx = openssl_verify_none();
    super::shared::BenchmarkResponse::ok(&format!("Connected to {} with {}", url, ctx))
}
fn openssl_verify_none() -> String { "openssl(verify=SSL_VERIFY_NONE)".to_string() }
